src/Utils/api.js

```diff
@@ -55,3 +55,25 @@ export const postComment = (article_id, newComment) => {
       console.log(error);
     });
 };
+
+export const getTopics = () => {
+  return api
+    .get("/topics")
+    .then((res) => {
+      return res.data.topics;
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
In case you want to indicate any error messages, instead of logging here, better to display a message to the used when something goes wrong.